### ISSUE TITLE
Gradient selector and color links lack ARIA labels

### ISSUE THREAD

**Issue by SteelWagstaff** [Automattic/Iris#69]
We recently worked with the Inclusive Design Research Centre to conduct an accessibility audit on our WordPress-based software backend (for [Pressbooks](https://pressbooks.org/)). In the course of that audit, the auditor indicated that the content of the color picker is not communicated to users of assistive technologies. For example, the selector within the gradient is communicated as `link,` and the various colors (black, white, red, etc.) are announced as `group`. 
![Screen Shot 2019-04-29 at 1 13 03 PM](https://user-images.githubusercontent.com/13485451/56917221-8d455480-6a80-11e9-8e30-748b733b76ee.png)

Our auditors indicated that this can be fixed by adding [an ARIA label](https://www.w3.org/TR/WCAG20-TECHS/ARIA6.html) to each color so that it says something other than 'group' (i.e. the color's name or hex value).
